packages/mobile/src/account/BankAccounts.test.tsx

```diff
@@ -89,6 +89,17 @@ describe('BankAccounts', () => {
     expect(deleteFinclusiveBankAccount).toHaveBeenCalled()
     expect(ValoraAnalytics.track).toHaveBeenCalledWith(CICOEvents.delete_bank_account, { id: 2 })
   })
+  it('shows a loading circle when getFinclusiveBankAccounts is inflight', async () => {
+    const { getByTestId, queryByTestId } = render(
+      <Provider store={store}>
+        <BankAccounts {...mockScreenProps} />
+      </Provider>
+    )
+
+    expect(getByTestId('Loader')).toBeTruthy()
+    await waitFor(() => expect(getFinclusiveBankAccounts).toHaveBeenCalled())
+    await waitFor(() => expect(queryByTestId('Loader')).toBeFalsy())
```

Comment:
if you added an assertion that `AddAccount` is visible here it would probly bring the patch coverage up to where it's supposed to be